packages/core/http/core-http-router-server-internal/src/versioned_router/core_versioned_route.ts

```diff
@@ -100,8 +103,8 @@ export class CoreVersionedRoute implements VersionedRoute {
   }
 
   /** This method assumes that one or more versions handlers are registered  */
-  private getDefaultVersion(): ApiVersion {
-    return resolvers[this.router.defaultHandlerResolutionStrategy]([...this.handlers.keys()]);
+  private getDefaultVersion(strategy: Exclude<HandlerResolutionStrategy, 'none'>): ApiVersion {
+    return resolvers[strategy]([...this.handlers.keys()]);
```

Comment:
Probably a naive question, but why not implementing a `none` strategy and allowing them to return `ApiVersion | undefined` or something?

Working with `Exclude<HandlerResolutionStrategy, 'none'>` seems like it could be a code smell?

Reply:
Good point, can clean up the code a little bit by not doing this.